api/src/main/resources/public/api/schema/v0/jobspec_v0.schema.json

```diff
@@ -0,0 +1,239 @@
+                "description": "Cache fetched artifact if supported by Mesos fetcher module"
+              }
+            },
+            "required": [ "uri" ]
+          }
+        },
+        "cmd": {
+          "description": "The command that is executed.  This value is wrapped by Mesos via `/bin/sh -c ${app.cmd}`.  Either `cmd` or `args` must be supplied. It is invalid to supply both `cmd` and `args` in the same app.",
+          "type": "string",
+          "minLength": 1
+        },
+        "cpus": {
+          "type": "number",
+          "description": "The number of CPU shares this application needs per instance. This number does not have to be integer, but can be a fraction.",
+          "minimum": 0
+        },
+        "disk": {
+          "type": "number",
+          "description": "How much disk space is needed for this application. This number does not have to be an integer, but can be a fraction.",
+          "minimum": 0
+        },
+        "docker": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "image": {
+              "type": "string",
+              "documentation": "The docker repository image name."
+            }
+          },
+          "required": ["image"]
+        },
+        "env": {
+          "type": "object",
+          "patternProperties": {
+            ".*": {
+              "oneOf": [
+                { "type": "string" }
+              ]
+            }
+          }
+        },
+        "maxLaunchDelay": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "The number of seconds until the job needs to be running. If the deadline is reached without successfully running the job, the job is aborted."
+        },
+        "mem": {
+          "type": "number",
+          "description": "The amount of memory in MB that is needed for the application per instance.",
+          "minimum": 0
+        },
+        "placement": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "constraints": {
+              "type": "array",
+              "description": "The array of constraints to place this job.",
+              "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "attribute": {
+                    "type": "string",
+                    "description": "The attribute name for this constraint."
+                  },
+                  "operator": {
+                    "type": "string",
+                    "description": "The operator for this constraint.",
+                    "enum": ["EQ", "LIKE", "UNLIKE"]
+                  },
+                  "value": {
+                    "type": "string",
+                    "description": "The value for this constraint."
+                  }
+                },
+                "required": ["attribute", "operator"]
```

Comment:
@meichstedt thank you very very much. and i see with your reply. thank you. but i am sorry that  i am looking for he ability to use "unique" for the goal. the goal is that a new job will not launch if there is an existing run, and the new job will launch if there is not an existiong run or after the previous job finishes. and i try "concurrencyPolicy", but it is only support “ALLOW”. 
could you help me? thanks very very much.